src/main/resources/application-prod.yml

```
@@ -154,4 +154,8 @@ ai:
 
 #운영환경 도메인
 app:
-  base-url: ${BASE_URL} # 운영 환경 도메인
\ No newline at end of file
+  base-url: ${BASE_URL} # 운영 환경 도메인
+
+
+cors:
+  allowed-origins:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)